DRAFT - blog 2025 review from James (#14256)
PostHog/posthog.com: contents/blog/2025-review.md

---
date: 2025-12-31
title: 'How 2025 went for PostHog'
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
category: CEO diaries
author:
  - james-hawkins
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-ceo-diary-blog.png
featuredImageType: full
---

> It's December 31st, 2025, and I've decided to hang out in a near-empty WeWork to write about how the year went. I want to clear my head before next year starts, and thought it make for an interesting read for others.

Reviewer: ```suggestion
It's December 31st, 2025, and I've decided to hang out in a near-empty WeWork to write about how the year went. I want to clear my head before next year starts, and thought it'd make for an interesting read for others.
```

Here are my reflections:

> ## Money

Reviewer: I think this section is quite a bit beyond just money. The AI-inference margin point is great. 
```suggestion
## Money and growth
```

This growth is outrageously fast, but the funny thing is that this doesn't sound amazing compared to other companies going from $0 to $100M in a year. This has genuinely been a challenge to handle - "is this FOMO or a good idea for _our_ users" is a criticism I've received this year.

Recently I went on a short trip to Dubai with some of our investors and I chatted about this phenomen with Arnav from PeakXV. He said he could see a world where people look at AI-inference heavy companies as a different business model to SAAS, where it's acceptable to have a gross margin 25% of what has been standard in SAAS, but thus you just need to grow 4 times faster. Net profit growth being the metric, not "ARR" growth. It'll be interesting to see how we handle this through 2026 as we ship a bunch of AI first products and features.

> Long ago we realized that our growth was largely from word of mouth. This comes out as a _lot_ of organic search volume, but mostly for the word "PostHog". We hired a friend and former colleague for a year or so and he went one level deeper - on finding out a user was recommended PostHog, he asked them _why_. They came back with four reasons: lots of products in one, the lowest prices, developer brand and a very technical and helpful customer service. We've been trying to do more and more of those things over time to grow.

Reviewer 2: I'm confused about who we hired for a year or so, why they left, and why that's relevant to this sentence.

We raised a ton of money too - $145M of capital for the business through our Series D and E this year. Psychologically this has been a big shift. We wouldn't have known how to reliably spend money to grow in 2023, we started to get an idea through 2024 and this was the year we acted on it. It means that our constraint has shifted from confidence to capital to our ability to find really strong engineers. That's a pretty strange feeling and a little unnerving. It can feel frustrating at times - we have _so_ many things to build at once that we believe in. I am very glad we are able to hire across so many countries. I couldn't imagine trying to do this in just commuting distance of one or two offices.

> One thing we are realizing about hiring - we get lots of applications for the roles that we market to. We haven't really done much outbound recruitment at all. We write all the time about product engineering - we have a [newsletter](https://newsletter.posthog.com/) on this topic with over 100K subscribers, and we build tools for product engineers. However, when it comes to finding different engineering expertise - such as deeper infrastructure stuff - applicants are understandably rarer. We want to figure out outbound recruitment in 2026 to make sure we get people with some of the more specialist skills we need as we keeping scaling our stack.

Reviewer: I didn't quite understand what you were saying, but I think this makes more sense
```suggestion
One thing we are realizing about hiring - we're going to need to do outbound recruitment for some roles. We traditionally haven't needed to do this as we market to the same audience we get applicants from - we build tools for product engineers and have a [newsletter](https://newsletter.posthog.com/) about this with over 100K subscribers. However, when it comes to finding different engineering expertise - such as deeper infrastructure stuff - applicants are understandably rarer. We want to figure out outbound recruitment in 2026 to make sure we get people with some of the more specialist skills we need as we keeping scaling our stack.
```

This is still very much the start - we've far more we're going to implement here too. I am confident that it will be mindblowing.

### We got told we are too complicated, many times

We monitor our brand mentions online. When we look at _negative_ feedback, the most popular criticism is that the UX can be overwhelming. PostHog has nearly 20 products, so that's not surprising. However, it's increasingly an issue as we build more products and add more features to those we already have.

The mental model I used to hold of PostHog was similar to that I have of AWS - it is hundreds of products, you simply ignore 95% of them. They're all reliable and have most of the functionality of the competitors so you just pick them by default each time since you already have an account and it's often better to use them together. It's complex but you ignore most of it.

> AI has changed this view - we've already demonstrated that agents can use our products together. My personal use of PostHog is now AI first.

Reviewer: Trying to square the circle of why AI means simpler, there's something slightly confusing here. 
```suggestion
AI has changed this view - it means you will be able to use all our products together and be simpler. My personal use of PostHog is now AI first.
```

Combining this with our existing, but nascent, workflows product feels like it could unlock a combinatorial effect in how many things we can achieve for our customers.

Helping provide more of the tracking and infrastructure for agents is the other key area to explore. So many people are building agents, but making it easier to spin them up, understand their behavior and improve them feels like a dark art still. We have shipped LLM analytics already, but it's largely prompt-focused. We think extending our functionality for agents would make a ton of sense if we want to be where the puck is going.

##  We hired lots of people

Surprisingly this went super smoothly. We felt very confident that we'd overengineered the living daylights out of our culture, and that's pretty much what happened. The handbook was a _huge_ deal to have in place. We changed a lot of stuff, but we had remarkably few issues with adding far more people and more teams. We went from around ~80 people at the end of 2024 to 179 ish by the end of 2025.

## We think 2026 will be one of our most challenging years

> 2026 is the year we go from being the underdog to being the same order of magnitude in terms of revenue. We have a far bigger customer base than many of them in terms of numbers.

Reviewer: underdog to who? customer base compared to who? 😅
